remerkleable/bitfields.py

```diff
@@ -326,6 +326,8 @@ def serialize(self, stream: BinaryIO) -> int:
 
     @classmethod
     def navigate_type(cls, key: Any) -> Type[View]:
+        if key == '__len__':
+            return int
```

Comment:
I think returning `uint256` may be better. The main idea of `navigate_type` is to get a type that can be used to wrap around the underyling "backing", the merkle tree that is interpreted through typed views.

Also, adding a simple test-case of navigating to a `__len__` node, and then reading it by doing the generalized merkle-tree lookup & typed interpretation of the value, would be great.

Reply:
yes, just found this: https://github.com/protolambda/remerkleable/blob/91ed092d08ef0ba5ab076f0a34b0b371623db728/remerkleable/union.py#L174

I will update!